### ISSUE TITLE
Broken input in friendz_android

### ISSUE THREAD

**Issue by privat** [privat/nit#848]
Since c710bb4a73ea9de8fdfe5e8633eac557b788abd2 (I bisected) the input of friendz in android if off.

The program does the following

~~~
var w = screen_width
display.set_viewport(0,0,w,w*display.height/display.width)
~~~

By commenting it, the board is displayed as a post-stamp but the input works.

I conclude that Viewport does not seem to affect the input as one could expect :(

**Comment by privat** [comment]
is this one closed by #1076 ?

**Comment by xymus** [comment]
Yep!
